### PR TITLE
user service changes

### PR DESCRIPTION
**Pull request by Saikumar2501**
user service changes

### CODE REVIEW

File: src/main/java/com/pst/user/controller/UserController.java [hunk 1]
```diff
@@ -1,46 +1,58 @@
...
-	public String hello() {
-		System.out.println("hello calling...");
-		return "hello";
-	}
+	@Autowired
+	UserService service;
 	
 	@PostMapping
-	public void addUser() {
+	public UserEntity addUser(@RequestBody UserEntity user ) {
```
Comment (phanisofttech) [inline src/main/java/com/pst/user/controller/UserController.java]: Remove extra space

**Review by phanisofttech — CHANGES_REQUESTED** [review]

File: src/main/java/com/pst/user/controller/UserController.java [hunk 2]
```diff
@@ -1,46 +1,58 @@
 package com.pst.user.controller;
 
+import java.util.List;
+import java.util.Optional;
+
```
Comment (Charan9701) [inline src/main/java/com/pst/user/controller/UserController.java]: Remove extra spaces between imports

File: src/main/java/com/pst/user/controller/UserController.java [hunk 3]
```diff
@@ -1,46 +1,58 @@
...
+	@GetMapping("/all-users")
+	public List<UserEntity> viewAllUsers() {
+		
+	  return service.getAllUsers();
 		
 	}
 	
 	@DeleteMapping("{adharNumber}")
-	public void deleteUser(@PathVariable("adharNumber") long adharNumber) {
+	public String deleteUser(@PathVariable("adharNumber") long adharNumber) {
```
Comment (Charan9701) [inline src/main/java/com/pst/user/controller/UserController.java]: Change the adharNumber spelling to aadhaarNumber

**Review by Charan9701 — CHANGES_REQUESTED** [review]
Change the spelling of adharNumber in database to aadhaarNumber

**Pull request merged**